### PR TITLE
Fix missing titles on 48in24 cards

### PR DESCRIPTION
**Pull request by dem4ron**

### CODE REVIEW

File: app/views/challenges/48in24.html.haml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 - week = ((Time.zone.today - Date.new(2024, 1, 15)) / 7).ceil
-- featured_exercise = @exercises.find { |e| e.week == week }
+- exercises = @exercises.select { |e| e.week <= week }
```
Comment (ErikSchierboom) [inline app/views/challenges/48in24.html.haml:2]: I like the old naming a bit better

```suggestion
- live_exercises = @exercises.select { |e| e.week <= week }
```

File: app/views/challenges/48in24.html.haml [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+- tracks = Track.where(slug: exercises.flat_map(&:featured_tracks)).pluck(:slug, :title).to_h
+
+- featured_exercise = exercises.last
 - if featured_exercise
   - generic_exercise = GenericExercise.for(featured_exercise.slug)
-- live_exercises = @exercises.select { |e| e.week <= week }
-- flat_live_exercises = live_exercises.flat_map(&:featured_tracks).uniq
-- featured_tracks = Track.where(slug: flat_live_exercises).order(:title).pluck(:slug, :title).to_h
-- num_placeholders = 48 - live_exercises.size
+  - featured_tracks = Track.where(slug: featured_exercise.featured_tracks).pluck(:title).sort
```
Comment (ErikSchierboom) [inline app/views/challenges/48in24.html.haml]: Could you check to see if this works? It saves a query
```suggestion
  - featured_tracks = featured_exercise.featured_tracks.select{|slug| tracks[slug]}.sort
```

File: app/views/challenges/48in24.html.haml [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+- num_placeholders = 48 - exercises.size
```
Comment (ErikSchierboom) [inline app/views/challenges/48in24.html.haml]: Could this be?

```suggestion
- num_placeholders = @exercises.size - exercises.size
```

File: app/views/challenges/48in24.html.haml (hunk 2 above)
Comment (dem4ron) [reply app/views/challenges/48in24.html.haml]: Yes, this works. :)

File: app/views/challenges/48in24.html.haml (hunk 1 above)
Comment (iHiD) [reply app/views/challenges/48in24.html.haml:2]: Happy to change it back. It had become messy in the interviening commits. @dem4ron  could you do that pls?

**Comment by iHiD** [comment]
@dem4ron Could you follow up with https://github.com/exercism/website/pull/6733#discussion_r1476551508 pls?

**Pull request merged**